### ISSUE TITLE
Buggy easylist matching?

### ISSUE THREAD

**Issue by ghostwords** [sammacbeth/isitblocked#76]
Hello!

A URL/domain that I expect to be blocked by EasyPrivacy is reported as not blocked.

```
$ curl -s "https://easylist.to/easylist/easyprivacy.txt" | grep rfksrv.com
||rfksrv.com^$third-party
@@||rfksrv.com^$script,domain=riteaid.com
```

```
$ ./cli.js -u "https://244631299-prod.rfksrv.com/rfk/js/12359-244631299/init.js"

  ✔ Fetch and load block lists
  ❯ Test URL https://244631299-prod.rfksrv.com/rfk/js/12359-244631299/init.js
    ✔ Cliqz AntiTracking
    ✔ Cliqz Adblocker (ads)
    ✔ Ghostery
    ✔ Disconnect
    ✔ easylist
    ✔ easyprivacy
    ✔ peterlowe
    ✖ Adguard DNS
      → Blocked: 244631299-prod.rfksrv.com
    ✖ DuckDuckGo
      → block: matched rule - block
```
```
$ ./cli.js -h rfksrv.com

  ✔ Fetch and load block lists
  ❯ Test URL https://rfksrv.com/
    ✔ Cliqz AntiTracking
    ✔ Cliqz Adblocker (ads)
    ✔ Ghostery
    ✔ Disconnect
    ✔ easylist
    ✔ easyprivacy
    ✔ peterlowe
    ✖ Adguard DNS
      → Blocked: rfksrv.com
    ✖ DuckDuckGo
      → block: matched rule - block
```

**Comment by sammacbeth** [comment]
Thanks for this report! You're correct, it looks like `$third-party` rules were not being matched correctly. This was because we were not passing a request `type` to the adblocker matching, which meant that the `adblocker` library [defaulted to `main_frame`](https://github.com/cliqz-oss/adblocker/blob/master/packages/adblocker/src/request.ts#L243). The consequence of this is that matching always consider the request as first party.

I've fixed this in #80 and will prepare a new release with the fix. I'll also expose the `sourceUrl` and `type` options via the CLI so for rules where this is important they can be explicitly provided.

**Comment by sammacbeth** [comment]
Fixed in https://github.com/sammacbeth/isitblocked/releases/tag/v2.0.0
